test/integration/components/sqlclient/gosqlclient.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
```

Comment:
Just thinking out loud here, wouldn't it be simpler to use an in memory SQL backend for this test? The Go SQL package in vendor neutral, it might be much easier to write a test with sqlite instead of with postgres.

Reply:
I agree. Sqlite is pretty simple to embed. It should be as simple as:

```go
import _ "github.com/mattn/go-sqlite3"
``` 

(or any other driver listed in the [official wiki](https://github.com/golang/go/wiki/SQLDrivers))

Reply:
I went with modernc.org/sqlite